crates/orchestrator/src/jobs/state_update_job/mod.rs

```diff
@@ -226,10 +215,18 @@ impl StateUpdateJob {
             let onchain_data_size = 0;
             settlement_client.update_state_calldata(program_output, onchain_data_hash, onchain_data_size).await?
         } else if snos.use_kzg_da == Felt252::ONE {
-            // TODO: Build the blob & the KZG proof & send them to update_state_blobs
-            let kzg_proof = self.fetch_kzg_proof_for_block(block_no, fetch_from_tests).await;
-            let kzg_proof: [u8; 48] = kzg_proof.try_into().expect("kzg proof size must be 48 bytes");
-            settlement_client.update_state_blobs(vec![], kzg_proof).await?
+            let blob_data = fetch_blob_data_for_block(block_no).await?;
+            let x_0_value = fetch_x_0_value_from_os_output(block_no).await?;
```

Comment:
Now I actually think this will become `fetch_snos_output`. It doesn't make sense to only store x0 in a file. wdyt?

Reply:
I think it is easier to read if just store the x_0 otherwise we would need to go line by line and at certain index read the x_0 point. So I was thinking that what if we store the x_0 separately after SNOS run.

Reply:
hmm. so i was thinking we do the parsing and everything at the SNOS job itself (the same way we assume it right now) and store a json which has fields like 

```json
{
  "pie": <PIE>,
  "x0": x0
}
```

Reply:
basically, avoiding multiple files

Reply:
this approach also works.